src/components/TopBar/components/UserMenu/tests/UserMenu.test.tsx

```diff
@@ -17,13 +16,37 @@ describe('<UserMenu />', () => {
   };
 
   it('renders with the given props', () => {
-    const userMenu = mountWithAppProvider(<UserMenu {...userMenuProps} />);
-    expect(userMenu.find(UserMenu).props()).toStrictEqual(userMenuProps);
+    const userMenu = mountWithApp(<UserMenu {...userMenuProps} />);
```

Comment:
This is the only big change that I am doing, but not sure if this makes sense to test that the props are being render.

Reply:
This makes sense to me